### DISCUSSION TITLE
Discussion on the formation and voting process for the TSC

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#52] [General]
The FINOS Governing Board have approved the formation of the TSC. An important next step is to determine the more permanent make-up of the TSC and how we achieve that.

This discussion thread re-opens a conversation I started on the [FINOS Community](https://github.com/finos/community/issues/162) project:

This process draws on prior art, having considered the similar committees within similar organisations (e.g. [CNCF TOC](https://github.com/cncf/toc), [Hyperledger TSC](https://tsc.hyperledger.org/), [LFN TAB](https://linuxfoundation.org/technical-advisory-board/) …)

For the TSC to be a success, we need members that are:

- Engaged, and active - committing time to both formal meetings and various other activities. They should actively engage with FINOS Projects.
- Knowledgeable and professional
- Able to leverage their organisation
- Neutral - first-and-foremost representing the needs of the members

In other organisations it is common for the TSC (or similar) to have seats allocated to different classes of vote. This reflects the [FINOS GB](https://www.finos.org/governing-board), where the different membership levels (platinum, gold, silver) have different levels of elected representation. It is proposed that with the TSC we do the same.

The TSC would have 13 seats, each having a two year term
 - 6 elected by the FINOS Governing Body 
 - 7 elected by the Community

Nominations would be encouraged from both the community and the GB. Nominees must either be active contributors (as defined by LFX Insights) or an employee of a member organisation. FINOS already has a process for collecting nominations, and tallying votes (which is used for GB representation), we would use the same process and mechanisms for the TSC.

The voting process would have two rounds:
 - The FINOS GB vote would be conducted and 6 individuals would be elected
 - Following this, of the remaining candidates, a community vote (involving active contributors) would be held, with the 7 remaining seats filled.

Following the voting process, the TSC would meet to elect a Chair, with the initial meetings focussed on elaborating the TSC responsibilities and formalising this as part of the overall [FINOS Governance](https://www.finos.org/governance).

After community consultation, the goal is to bring a proposal to the Membership & Governance Committee (a group that assists the GB in governance related matters) in March and the Governing Body in April.

Through the pilot, we have have that the level of engagement from some members has been very high, while for others it has been comparatively low. I also think that we should formalise the level of commitment we expect from TSC members. I'd propose that if a member doesn't attend 50% or more meetings, this triggers a conversation with the Chair, who may decide that they should step-down, with their seat filled by another suitable candidate.

There are clearly some more details required in the above, but I'd be interested in initial opinions on the overall shape outlines above.

**Comment by stephengoldbaum** [comment]
Looks solid.  50% still seems low. Perhaps we can aim higher with an option for using delegate when needed.

**Reply by jonfreedman** [reply]
Agree, I think rather than just a percentage we probably should talk about consecutive meetings as well. If we settle on a monthly cadence and someone skips two in a row without advance apologies that's not great.
